plugins/modules/lvol.py

```diff
@@ -75,12 +75,12 @@
     - Can be used to create a logical volume, hence when I(state=present).
     type: int
     default: 1
-  num_of_logical_partitions:
+  size:
     description:
     - Specifies the number of logical partitions or the size of the
       the logical volume in terms of K, M, or G.
     - Can be used to create a logical volume, hence when I(state=present).
-    type: int
+    type: str
```

Comment:
if the new type for size is str, is the default meant to stay as 1?

Reply:
in ansible under the argument_spec in the main function
if the parameter type is set to str
any non str user input will be converted to string
i.e. if i input 
1 it will be converted to "1" when it receives the user input
although "1" is more clear, inputting 1 will work as well

Reply:
ok cool, thanks for clarifying.